Owl/README.md

```diff
@@ -3,19 +3,18 @@
 <img src="screenshots/collage_header.png" alt="Owl collage"/>
 
 ## Introduction
-Owl is an educational app that uses Material Design components and [Material Theming](https://material.io/design/material-theming) to create an energetic, motivational brand experience.
-This project is an Android implementation of [Owl](https://material.io/design/material-studies/owl.html), a Material study meant to showcase componentry and theming using the [Material Components for Android library](https://github.com/material-components/material-components-android).
+This project is an Android implementation of [Owl](https://material.io/design/material-studies/owl.html), a Material Study showcasing the possibilities of using Material Theming and Material Components for Android.
 
 ## Screenshots
 <img src="screenshots/owl_demo.gif" height="400" alt="Owl sample demo"/>
 
 ## Material Theming
-Owl employs material theming to reflect the energy and excitement of learning a new skill, using a bold aesthetic and unfilled shapes that invite the user to populate them with new content and courses. The app customizes [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
+Owl uses Material Theming to customize the app’s [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
```

Comment:
Remove `to achieve this` at the end of this sentence?

Reply:
Good catch! Agreed, remove "to achieve this"